rules/windows/privilege_escalation_printspooler_malicious_registry_modification.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/07/06"
+maturity = "production"
+updated_date = "2021/07/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects attempts to exploit privilege escalation vulnerabilities related to the Print Spooler service. For more
+information refer to the following CVE-2021-34527 and verify that the impacted system is investigated.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential PrintNightmare Exploit Registry Modification"
+references = [
+    "https://msrc.microsoft.com/update-guide/vulnerability/CVE-2021-34527",
+    "https://github.com/afwu/PrintNightmare"
+]
+risk_score = 73
+rule_id = "6506c9fd-229e-4722-8f0f-69be759afd2a"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+registry where process.name : "spoolsv.exe" and
+  user.id == "S-1-5-18" and
```

Comment:
Sid probably not necessary but doesn't hurt either

Reply:
IIRC there are some legit instances of spoolsv.exe not running as system.